### PR TITLE
Implement element_at, ignore_elements, refactor distinct

### PR DESCRIPTION
**Pull request by godhand4826**
Sorry, I force push when "Refactor skip_last operator".  (See #123)
I create  "Implement element_at, ignore_elements, refactor distinct" PR again.

### CODE REVIEW

**Comment by M-Adoo** [comment]
Can you update the changle log?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,16 @@
 ### Features
 - **operator**: add `distinct` operator.
 - **operator**: add `debounce` operator.
+- **operator**: add `element_at` operator.
```
Comment (M-Adoo) [inline CHANGELOG.md:7]: Should add the features and refactor below the `Unreleased`,  not add in a released version `0.10.0.

**Review by M-Adoo — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (godhand4826) [reply CHANGELOG.md:3]: Okay

Comment (godhand4826) [reply CHANGELOG.md:3]: Done

**Review by M-Adoo — APPROVED** [review]
Thanks!

**Pull request merged**